Resources/Prototypes/Catalog/Fills/Backpacks/duffelbag.yml

```diff
@@ -336,3 +336,15 @@
       - id: ClothingHandsGlovesNitrile
       - id: SyringeTranexamicAcid
       - id: SyringeHyronalin
+
+- type: entity
```

Comment:
this should be in the appropriate area in the DeltaV namespace:
Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/duffelbag.yml